### ISSUE TITLE
PK-Sim Version 9.0 stopped working

### ISSUE THREAD

**Issue by arketkar** [Open-Systems-Pharmacology/PK-Sim#1514]
Hi
Today I installed the new version 9 of PK-Sim. After installation and restarting my laptop, I tried to open the program, however it just crashed down immediately with the following message.
![image](https://user-images.githubusercontent.com/32693494/85429016-b5084d80-b59b-11ea-8014-8b2bf41ec6ac.png)
Then I uninstalled and re-installed the entire setup, however I faced the same problem. So I 'repaired' the setup through control panel. Still the same problem....its not opening at all. MoBi is opening though.

I had installed Version 8 on the same system and it was running fine. My system configuration is -
Windows 7 Professional with SP1, 64 bit OS; 4 GB RAM and 1 TB hard disk.

Appreciate if you could help resolve this issue.

Thanks and regards.
Anant

**Comment by Yuri05** [comment]
@arketkar Can you export and attach you Error event log again?

After that, please try to increase the number of allowed GDI objects as desscribed [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/862#issuecomment-466048852) and restart your computer.
I doubt that this will solve your problem however.

**Comment by arketkar** [comment]
Hi @Yuri05  I had attached the new Error event report earlier. Attaching that again for your ready reference (generated today).

[Error log_24062020_arketkar.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/4825684/Error.log_24062020_arketkar.zip)

I shall follow the additional steps recommended by you and revert. Thanks.

**Comment by arketkar** [comment]
Hi @Yuri05  I tried the steps mentioned in the link provided by you. Sorry to say but its not working. It just opened the project file, I could run one DDI simulation. The moment I clicked on export 'PK-Analysis to csv' button it again crashed down.....

@msevestre Version 8 was running fine on the same machine...however running Version 9 is becoming a problem.

Appreciate if your team can resolve this issue.

Thanks again in advance.

**Comment by msevestre** [comment]
@arketkar v9 is using a newer version of the .NET framework for example. We have no problem running the setup on win7 machines and all our tests were also successful. I am not hopeful but @Yuri05 may have other ideas :)

Just to try something .Could you download the portable version of pksim? and see if this works?

https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/download/v9.0/pk-sim-portable-setup.zip

e.g. download, unzip etc..

**Comment by arketkar** [comment]
Hi @msevestre thanks for sharing the link. I tried the portable version as well. However its not working on my system...

Can you please elaborate more on the newer version of the .NET framework? Can I request my IT team to upgrade/change something in my system so that the system would become suitable to run Version 9. Is there a way I could find out the version of .NET framework and inform you?

Sorry to bother you and your team members...

**Comment by Yuri05** [comment]
@arketkar Looking at your latest EventLog - it seems that the .NET Framework **4.7.2** (which is used in OSP9) does not work properly on your computer. E.g. below you can see a crash in the _WinForms FileDialog_ (which is one of the basic graphical elements). 
<details><summary>WinForms FileDialog crash</summary>

```
Application: PKSim.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 000007FEFD0CB3DD
Stack:
   at System.Windows.Forms.FileDialogNative+IFileDialog.Show(IntPtr)
   at System.Windows.Forms.FileDialogNative+IFileDialog.Show(IntPtr)
   at System.Windows.Forms.FileDialog.RunDialogVista(IntPtr)
   at System.Windows.Forms.CommonDialog.ShowDialog(System.Windows.Forms.IWin32Window)
   at OSPSuite.UI.Services.DialogCreator.selectionFor(System.Windows.Forms.FileDialog, System.String, System.String, System.String, System.String, System.String)
   at OSPSuite.UI.Services.DialogCreator.AskForFileToSave(System.String, System.String, System.String, System.String, System.String)
   at PKSim.Infrastructure.Services.SimulationExportTask+<exportToFileAsync>d__28.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.__Canon ByRef)
   at PKSim.Infrastructure.Services.SimulationExportTask.exportToFileAsync(System.String, System.String, System.String, System.Func`2<System.String,System.Threading.Tasks.Task>, System.String)
   at PKSim.Infrastructure.Services.SimulationExportTask.ExportPKAnalysesToCSVAsync(PKSim.Core.Model.PopulationSimulation)
   at PKSim.Presentation.UICommands.ExportPopulationSimulationPKAnalysesCommand.<PerformExecute>b__2_0(PKSim.Core.Services.ISimulationExportTask)
   at OSPSuite.Core.Extensions.TaskExtensions+<SecureAwait>d__2`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.__Canon ByRef)
   at OSPSuite.Core.Extensions.TaskExtensions.SecureAwait[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.__Canon, System.Func`2<System.__Canon,System.Threading.Tasks.Task>)
   at PKSim.Presentation.UICommands.ExportPopulationSimulationPKAnalysesCommand+<PerformExecute>d__2.MoveNext()
   at System.Runtime.CompilerServices.AsyncVoidMethodBuilder.Start[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.__Canon ByRef)
   at PKSim.Presentation.UICommands.ExportPopulationSimulationPKAnalysesCommand.PerformExecute()
   at OSPSuite.Presentation.UICommands.ObjectUICommand`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Execute()
   at OSPSuite.Presentation.UICommands.UICommandExtensions.ExecuteWithinExceptionHandler(OSPSuite.Presentation.MenuAndBars.IUICommand, OSPSuite.Utility.Events.IEventPublisher, OSPSuite.Core.Services.IChangePropagator)
   at OSPSuite.Presentation.UICommands.UICommandExtensions.ExecuteWithinExceptionHandler(OSPSuite.Presentation.MenuAndBars.IUICommand)
   at OSPSuite.Presentation.MenuAndBars.MenuBarButton.Click()
   at OSPSuite.UI.Mappers.MenuBarButtonToBarItemMapperBase+<>c__DisplayClass1_0.<UpdateBarButtonItem>b__0(System.Object, DevExpress.XtraBars.ItemClickEventArgs)
   at DevExpress.XtraBars.BarItem.OnClick(DevExpress.XtraBars.BarItemLink)
   at DevExpress.XtraBars.BarButtonItem.OnClick(DevExpress.XtraBars.BarItemLink)
   at DevExpress.XtraBars.BarItemLink.OnLinkClick()
   at DevExpress.XtraBars.BarButtonItemLink.OnLinkAction(DevExpress.XtraBars.BarLinkAction, System.Object)
   at DevExpress.XtraBars.ViewInfo.BarSelectionInfo.ClickLink(DevExpress.XtraBars.BarItemLink)
   at DevExpress.XtraBars.ViewInfo.BarSelectionInfo.UnPressLink(DevExpress.XtraBars.BarItemLink)
   at DevExpress.XtraBars.Ribbon.Handler.BaseRibbonHandler.OnUnPressItem(DevExpress.Utils.DXMouseEventArgs, DevExpress.XtraBars.Ribbon.ViewInfo.RibbonHitInfo)
   at DevExpress.XtraBars.Ribbon.Handler.BaseRibbonHandler.OnMouseUp(DevExpress.Utils.DXMouseEventArgs)
   at DevExpress.XtraBars.Ribbon.Handler.RibbonHandler.OnMouseUp(DevExpress.Utils.DXMouseEventArgs)
   at DevExpress.XtraBars.Ribbon.RibbonControl.OnMouseUp(System.Windows.Forms.MouseEventArgs)
   at System.Windows.Forms.Control.WmMouseUp(System.Windows.Forms.Message ByRef, System.Windows.Forms.MouseButtons, Int32)
   at System.Windows.Forms.Control.WndProc(System.Windows.Forms.Message ByRef)
   at DevExpress.Utils.Controls.ControlBase.WndProc(System.Windows.Forms.Message ByRef)
   at DevExpress.XtraBars.Ribbon.RibbonControl.WndProc(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr, Int32, IntPtr, IntPtr)
   at DevExpress.Utils.Drawing.Helpers.NativeMethods+UnsafeNativeMethods.DefSubclassProc(IntPtr, IntPtr, IntPtr, IntPtr)
   at DevExpress.Utils.Drawing.Helpers.NativeMethods+UnsafeNativeMethods.DefSubclassProc(IntPtr, IntPtr, IntPtr, IntPtr)
   at DevExpress.Utils.Drawing.Helpers.Win32SubclasserFactory+Win32Subclasser.SubClassProcInner(IntPtr, IntPtr, IntPtr, IntPtr, IntPtr, IntPtr)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG ByRef)
   at System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG ByRef)
   at System.Windows.Forms.Application+ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr, Int32, Int32)
   at System.Windows.Forms.Application+ThreadContext.RunMessageLoopInner(Int32, System.Windows.Forms.ApplicationContext)
   at System.Windows.Forms.Application+ThreadContext.RunMessageLoop(Int32, System.Windows.Forms.ApplicationContext)
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.OnRun()
   at PKSim.UI.BootStrapping.PKSimApplication.OnRun()
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.DoApplicationModel()
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.Run(System.String[])
   at PKSim.Program.Main(System.String[])
```
</details>

Nothing we can really do here. Check your computer for problems (outdated drivers, missing system updates, etc.). If nothing helps - try to figure out with your IT-people what causes the problem.

**Comment by arketkar** [comment]
Hi @Yuri05 Thanks for informing about the .NET framework version 7.4.2. I installed that on my system along with the latest version 7.8, and confirmed that its functional. Then again installed PK-Sim version 9. It is was running fine, till I clicked on the 'PK-Analysis to csv' button. Why this particular function is causing a problem? Otherwise it was running fine. Why repeatedly it is crashing with Export to csv function?

In addition to that what I noticed that in the portable version 9, I could see the button for Export PK-Analysis to xlsx' file, and it worked well there. Why this option was not seen in the installed version of PK-Sim 9?

Sorry for too many questions....

Thanks again.
Anant
